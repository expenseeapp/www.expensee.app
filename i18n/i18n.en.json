{
  "slogan": "ExpenSeeãƒ»A Smart Finance Assistant",
  "description": "A secure expense tracking app using voice and natural language.",
  "SiteSubject": "Expense Tracking, Voice Expense Tracking, Natural Language Expense Tracking, Photo Expense Tracking, AI Expense Tracking, GPT Expense Tracking, Emotion Tracking, Expense Tracking API, Automatic Expense Tracking",
  "SiteKeyWords": "Expense Tracking, Voice Expense Tracking, Natural Language Expense Tracking, Photo Expense Tracking, AI Expense Tracking, GPT Expense Tracking, Emotion Tracking, Expense Tracking API, Automatic Expense Tracking",
  "SiteDescription": "ExpenSee is expense tracking app which supports speak to track expenses, anytime, anywhere, and data securely and privately stored in iCloud.",
  "SpeakToTrack": "Speak to Track Expense",
  "TextToTrack": "Text to Track Expense",
  "MoodsLight": "Not Just The Numbers, Also The Moods",
  "SiriYourAssistant": "Siri as Finance Assistant",
  "PhotoToTrack": "Photo to Track Expense",
  "APIAutomation": "API for Automation",
  "contact": "Contact",
  "status": "Status",
  "document": "Document"
}
