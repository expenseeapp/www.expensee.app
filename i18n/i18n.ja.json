{
  "slogan": "ExpenSee・記帳は記憶の記録",
  "description": "音声と自然な言葉で記録できる安全な家計簿アプリ",
  "SiteSubject": "記帳、音声入力、自然な言葉での入力、写真での入力、AIによる入力、GPTによる入力、感情の記録、APIによる入力、自動入力",
  "SiteKeyWords": "記帳、音声入力、自然な言葉での入力、写真での入力、AIによる入力、GPTによる入力、感情の記録、APIによる入力、自動入力",
  "SiteDescription": "ExpenSeeは、いつでもどこでも音声で支出を記録し、データをiCloudに安全に保存できる家計簿アプリです。",
  "SpeakToTrack": "音声で記録、手作業の手間を省く",
  "TextToTrack": "自然な言葉での記録、一言で済む",
  "MoodsLight": "感情の赤・黄・緑信号、数字だけでなく、感情も記録",
  "SiriYourAssistant": "Siriパーソナルファイナンスアシスタント",
  "PhotoToTrack": "写真での記録",
  "APIAutomation": "HTTP API対応",
  "contact": "開発者に連絡",
  "status": "サービス状況",
  "document": "ドキュメント"
}
