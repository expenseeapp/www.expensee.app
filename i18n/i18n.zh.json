{
  "slogan": "ExpenSee・记账也是记忆",
  "description": "支持语音和自然语言记录的安全记账应用",
  "SiteSubject": "记账, 语音记账, 自然语言记账, 拍照记账, AI 记账, GPT 记账, 记录情绪, 记账 API, 自动记账",
  "SiteKeyWords": "记账, 语音记账, 自然语言记账, 拍照记账, AI 记账, GPT 记账, 记录情绪, 记账 API, 自动记账",
  "SiteDescription": "ExpenSee, 一款可以随时随地通过语音直接记录消费并将数据安全存储在iCloud中的记账应用",
  "SpeakToTrack": "语音记账, 告别繁琐手动输入",
  "TextToTrack": "自然语言记账, 记录只需一句话",
  "MoodsLight": "情绪红绿灯, 不只记录数字，也包含情绪",
  "SiriYourAssistant": "Siri 个人财务助理",
  "PhotoToTrack": "拍照记账",
  "APIAutomation": "HTTP API 支持",
  "contact": "联系作者",
  "status": "服务状态",
  "document": "文档"
}
