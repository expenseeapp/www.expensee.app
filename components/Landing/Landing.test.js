import React from 'react'
import i18n from 'i18n-js'
import renderer from 'react-test-renderer'
import eni18n from '../../i18n/i18n.en.json'
import LandingPage from './Landing'
import zhi18n from '../../i18n/i18n.zh.json'

it('renders correctly', () => {
  i18n.translations = {
    en: eni18n,
    zh: zhi18n,
    'zh-Hant': zhi18n,
  }
  i18n.locale = 'en'
  i18n.fallbacks = true
  const tree = renderer.create(<LandingPage />).toJSON()
  expect(tree).toMatchSnapshot()
})
