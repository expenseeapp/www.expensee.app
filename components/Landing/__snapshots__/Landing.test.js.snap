// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="NavContainer"
  >
    <div
      className="NavColumnRight"
    >
      <a
        href="https://www.producthunt.com/posts/giki-2?utm_source=badge-featured&utm_medium=badge&utm_souce=badge-giki-2"
        rel="noreferrer"
        target="_blank"
      >
        <img
          alt="Giki - Blogging, note-taking, ideas management, thoughts catching. | Product Hunt"
          className="ProductHunt"
          src="https://api.producthunt.com/widgets/embed-image/v1/featured.svg?post_id=272045&theme=dark"
        />
      </a>
      <div
        className="NavItem"
      />
      <div
        className="NavItem"
      >
        <a
          href="https://giki.app/discover"
        >
          Discover
        </a>
      </div>
      <div
        className="NavItem"
      >
        <a
          href="https://giki.app/about"
        >
          About
        </a>
      </div>
      <div
        className="NavItem"
      >
        <a
          href="https://giki.app/sign"
        >
          Free Trial
        </a>
      </div>
    </div>
  </div>
  <div
    className="Container"
  >
    <div
      className="Responsive IntroContainer"
    >
      <div
        className="IconContainer"
      >
        <div
          className="SloganContainer"
        >
          <p
            className="SloganText"
          >
            Write Anytime, Anywhere
          </p>
        </div>
      </div>
      <div
        className="StoreContainer"
      >
        <div
          className="Container"
        >
          <div
            className="ColumnContainer"
            style={
              Object {
                "width": "50%",
              }
            }
          />
          <div
            className="ColumnContainer"
            style={
              Object {
                "width": "50%",
              }
            }
          />
        </div>
      </div>
    </div>
    <div
      className="Responsive Iphone"
    >
      <div
        className="IphoneScreenshot"
      >
        <span
          style={
            Object {
              "background": "none",
              "border": 0,
              "bottom": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": "initial",
              "left": 0,
              "margin": 0,
              "opacity": 1,
              "overflow": "hidden",
              "padding": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": "initial",
            }
          }
        >
          <img
            alt="App screenshot"
            data-nimg="fill"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt="App screenshot"
              data-nimg="fill"
              decoding="async"
              loading="lazy"
              sizes="100vw"
              src="/_next/image?url=%2Fimages%2Fhome-screen.png&w=3840&q=75"
              srcSet="/_next/image?url=%2Fimages%2Fhome-screen.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2Fhome-screen.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2Fhome-screen.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2Fhome-screen.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Fhome-screen.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Fhome-screen.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Fhome-screen.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Fhome-screen.png&w=3840&q=75 3840w"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </span>
      </div>
    </div>
  </div>
  <div
    className="Container Footer"
  >
    <div
      className="Sep"
    />
    <div
      className="FooterInfoWrapper"
    >
      <div
        className="Credit"
      >
        Made with 
        <span
          className="Heart"
        >
          â™¥
        </span>
         by
         
        <a
          href="https://i.giki.app"
        >
          Minghe
        </a>
      </div>
      <div
        className="Contact"
      >
        <a
          href="mailto:h.minghe@gmail.com?subject=giki.app"
        >
          Contact
        </a>
      </div>
      <div
        className="Contact"
      >
        <a
          href="https://gikiapp.github.io/status"
        >
          Status
        </a>
      </div>
    </div>
  </div>
</div>
`;
